contracts/Lens/PoolLens.sol

```diff
@@ -302,4 +302,180 @@ contract PoolLens is ExponentialNoError {
                 underlyingPrice: priceOracle.getUnderlyingPrice(address(vToken))
             });
     }
+
+    /**
+     * @dev Struct with pending reward info for a market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct with reward distribution totals for a single reward token and distributor.
+     */
+    struct RewardSummary {
+        address distributorAddress;
+        address rewardTokenAddress;
+        uint256 totalRewards;
+        PendingReward[] pendingRewards;
+    }
+
+    /**
+     * @dev Struct used in RewardDistributor to save last updated market state
```

Comment:
I just noticed we have a period at the end of some sentences lines but not all. We probably should be including periods

Reply:
Fixed that